looker-open-source/extension-template-kitchensink
# Update package dependencies

- Update dependencies
  - Update to Styled Components 5 & TS4.1
  - Latest Babel
- Remove `css-loader` & `style-loader` (no longer needed)
- Update references from `ActionList` to `DataTable` from `@looker/components`
- Leverage `webpack-cli` for scripts
- Reorganized `package.json` dependencies a little (move @types & build infrastructure packages to `devDependencies`)
